src/docs/guides/luminus.md

```diff
@@ -0,0 +1,201 @@
+
+```bash
+lein new luminus helloworld +postgres +immutant
+```
+
+A new Luminus app will be provisioned for you in the `helloworld` directory with support for PostgreSQL as the database and configures [Immutant](https://github.com/immutant/immutant) as the web server, which is production-ready and optimized for Clojure applications.
+
+**Note:** If you use MySQL or another database, you can pass it as an option when trying to create a new app.
+
+### Run the Luminus App locally
+
+Open `dev-config.edn` and add your Postgres database URL like so:
+
+```clojure
+ :database-url "postgresql://username:password@127.0.0.1:5432/helloworld_dev"
+```
+
+- `username:password` is your database user and password.
+- `helloworld_dev` is the database you have created locally.
+
+
+Next, run `lein run migrate` to run the database migrations.
+
+Finally, run `lein run` to launch your app!
+
+Open your browser and go to `http://localhost:3000` to see the app.
+
+### Prepare Clojure Luminus App for deployment
+
+1. We need to add the `ceshire` library to our dependencies. `cheshire` is a popular JSON encoding/decoding library in Clojure.
+
+Open your `project.clj` file and ensure you have the following under `:dependencies`:
+
+```clojure
+...
+[cheshire "5.10.0"]
+```
+
+Run the command below in your terminal to ensure it is installed:
+
+```bash
+lein deps
+```
+
+2. Create a `nixpacks.toml` file in the root directory of the app. 
+
+The [nixpacks.toml file](https://nixpacks.com/docs/configuration/file) is a configuration file used by Nixpacks, a build system developed and used by Railway, to set up and deploy applications. 
+
+In this file, you can specify the instructions for various build and deployment phases, along with environment variables and package dependencies.
+
+Add the following content to the file:
+
+```toml
+# nixpacks.toml
+
+[start]
+cmd = "java -jar $(find ./target -name '*.jar' ! -name '*SNAPSHOT*') migrate && java -jar $(find ./target -name '*.jar' ! -name '*SNAPSHOT*')"
+
+```
+
+Here, we are specifically instructing Nixpacks to use the following command to start the app.
+
+The command searches for all `.jar` files in the `target` directory (where the standalone JAR file is located after the build), excludes any file with "SNAPSHOT" in its name, and passes the selected file to `java -jar` to run.
+
+It starts by running the JAR file with the `migrate` option to apply database migrations. Once migrations are complete, it reruns the JAR file to launch the application.
+
+## Deploy the Luminus App to Railway
+
+Railway offers multiple ways to deploy your Clojure app, depending on your setup and preference. 
+
+### One-Click Deploy from a Template
+
+If you’re looking for the fastest way to get started, the one-click deploy option is ideal.
+
+Click the button below to begin:
+
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/new/template/DsDYI2)
```

Comment:
Thoughts on renaming the service from `clojureluminus` to `Clojure Luminus` or just `Luminus`?

Reply:
`Clojure Luminus` makes sense. Will do

Reply:
Done